saleor/core/schedules.py

```diff
@@ -0,0 +1,77 @@
+from collections import namedtuple
+from datetime import datetime
+
+import pytz
+from celery.schedules import BaseSchedule
+from celery.utils.time import maybe_timedelta, remaining
+from django.db.models import Q
+
+schedstate = namedtuple("schedstate", ("is_due", "next"))
+
+
+class sale_webhook_schedule(BaseSchedule):
```

Comment:
Is it expected that you didn't use python format for classes?

Reply:
https://github.com/saleor/saleor/pull/10110#discussion_r914685430